pkg/apiserver/controllers/v1/alerts.go

```diff
@@ -115,13 +116,30 @@ func (c *Controller) CreateAlert(gctx *gin.Context) {
 	}
 
 	for _, alert := range input {
-		if len(alert.Decisions) == 0 {
-			decisions, err := csprofiles.EvaluateProfiles(c.Profiles, alert)
-			if err != nil {
-				gctx.JSON(http.StatusInternalServerError, gin.H{"message": err.Error()})
-				return
+		// if len(alert.Decisions) != 0 {
```

Comment:
Is it intented to remove this check ?
If so, delete it instead of commenting it

Reply:
fixed via https://github.com/crowdsecurity/crowdsec/pull/857/commits/638729fbc8dadecd5dc7a7b1dac3d3afa669d117